app/routes/bps-multiple.js

```diff
@@ -21,7 +21,24 @@ module.exports = [{
         bps2022Value: joi.number().empty('').allow(null).precision(2).greater(0).less(1000000000),
         bps2023Value: joi.number().empty('').allow(null).precision(2).greater(0).less(1000000000),
         bps2024Value: joi.number().empty('').allow(null).precision(2).greater(0).less(1000000000)
-      }).min(1),
+      }).min(1).error(errors => {
+        errors.forEach(err => {
+          switch (err.code) {
+            case 'object.min':
+              err.message = 'There needs to be a least one payment amount for a scheme year.'
+              break
+            case 'number.greater':
+              err.message = `The payment amount for a ${err.local.key.replace('bps', '').replace('Value', '')} needs to be greater than 0 and less than 1000000000.`
+              break
+            case 'number.less':
+              err.message = `The payment amount for a ${err.local.key.replace('bps', '').replace('Value', '')} needs to be greater than 0 and less than 1000000000.`
```

Comment:
There are several error scenarios not caught here.  For example if I put 22222222222222222222222222 as the value I get the below error.

![image](https://user-images.githubusercontent.com/42574977/110490441-3aab3980-80e8-11eb-995f-eaedae0a815f.png)

Can we add a catch all to the `default` that just says something generic that the value is invalid?